vhd/lib/libvhd.c

```diff
@@ -2717,7 +2717,7 @@ vhd_open(vhd_context_t *ctx, const char *file, int flags)
 		struct stat stats;
 		err = stat(file, &stats);
 		if (err == -1)
-			return -errno;
+			return errno ? -errno : -EINVAL;
```

Comment:
Worth a comment here that this is necessary because Coverity doesn't understand that stat() sets errno? Otherwise someone might later read the man page for stat() and "clean it up"

Reply:
Of course Synopsis might fix that as well and then it can be cleaned up.